integration/spark/src/main/scala/org/apache/spark/sql/secondaryindex/load/CarbonInternalLoaderUtil.java

```diff
@@ -304,14 +304,16 @@ public static boolean updateLoadMetadataWithMergeStatus(CarbonTable indexCarbonT
 
   /**
    * Method to check if main table and SI have same number of valid segments or not
-   *
+   * @param mainTableLoadMetadataDetails
```

Comment:
no need of this, just method explanation is enough, as variable name suggest what info is it.